ImportInDB.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+import sys
+import os
+import csv
+
+from Database import *
+
+class ImportInDatabase():
+    """
+    Read a file in a Gellish Expression Format.
+    Rearrange in expressions and Store expressions in a Gellish database.
+    """
+    def __init__(self, fname):
```

Comment:
This method is quite long (~60 lines). It would be more readable if it was split up in several methods.